notebooks/02_model/sar_deep_dive.ipynb

```diff
@@ -70,9 +70,9 @@
     "\n",
```

Comment:
<p>this is nice</p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/617/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>